src/Infiltration/ui/Game.tsx

```diff
@@ -91,11 +92,14 @@ export function Game(props: GameProps): React.ReactElement {
       setStage(Stage.Countdown);
       pushResult(false);
       Player.receiveRumor(FactionName.ShadowsOfAnarchy);
-      // Kill the player immediately if they use automation, so
-      // it's clear they're not meant to
-      const damage = options?.automated
-        ? Player.hp.current
-        : calculateDamageAfterFailingInfiltration(props.StartingDifficulty);
+      let damage = calculateDamageAfterFailingInfiltration(props.StartingDifficulty);
+      // Kill the player immediately if they use automation, so it's clear they're not meant to
+      if (options?.automated) {
+        damage = Player.hp.current;
+        setTimeout(() => {
+          SnackbarEvents.emit("You was hospitalized. Do not try to automate infiltration!", ToastVariant.WARNING, 5000);
```

Comment:
```suggestion
          SnackbarEvents.emit("You were hospitalized. Do not try to automate infiltration!", ToastVariant.WARNING, 5000);
```
Just a typo / grammar fix